src/app/autocomplete-input/autocomplete-input.component.ts

```diff
@@ -37,11 +39,21 @@ export class AutocompleteInputComponent {
         this.schemaKeysStoreService.forPath(state.path).filter(item => item.startsWith(state.query))
       );
     });
+    this.valueChange$
+      .debounceTime(500)
```

Comment:
actually `typeahead` already has this feature via `typeaheadWaitMs` input

See the source code: https://github.com/valor-software/ngx-bootstrap/blob/development/src/typeahead/typeahead.directive.ts#L339

Reply:
Yes but is for another case. We want to debounce the input emission. `typeaheadWaitMs` debounce s the option matching.